avro-converter/src/test/java/io/confluent/connect/avro/AvroDataTest.java

```diff
@@ -1985,11 +2077,31 @@ protected void assertFieldEquals(
         assertEquals(msg, expectedDouble, actualDouble, expectedDouble / 100.0d);
         break;
       default:
-        if (!expectedDef.equals(actualDef)) {
-          int x = 0;
-        }
         assertEquals(msg, expectedDef, actualDef);
         break;
     }
   }
+
+  protected void assertValueEquals(Object expected, Object actual) {
+    if (actual instanceof byte[]) {
+      actual = ByteBuffer.wrap((byte[]) actual);
+    }
+    if (expected instanceof byte[]) {
+      expected = ByteBuffer.wrap((byte[]) expected);
+    }
+    assertEquals(expected, actual);
+  }
```

Comment:
Again, Avro doesn't seem to be consistent about when a `ByteBuffer` is required (for default values in an Avro schema's field) or not allowed (for values for schema parameters). Default values are pulled from both, so this method is needed to handle this inconsistency without failing.